utils/custom_properties.go

```diff
@@ -65,6 +65,14 @@ func appendCustomFields(metadata models.Metadata, customFields map[string]interf
 	return properties, err
 }
 
+func ConvertStringListToInterface(s []string) []interface{} {
```

Comment:
I thought it might be helpful incase we later need to use MapToProto in other places, so kept it generic like abduh suggested last time we were refactoring for the sqlutils package.

But since here we are using it only once, moving it to Postgres.